deploy/vagrant/bin/run_docker.sh

```diff
@@ -23,7 +23,7 @@ vagrant up --provider=docker --no-provision
 HOSTS=`vagrant ssh-config 2>/dev/null |grep -w Host |awk '{print $2}'`
 for h in $HOSTS; do
  addr=`vagrant ssh $h -c "ifconfig eth0" |grep -w inet|awk '{print $2}'`
- echo ${addr} ${h}".local" >> files/hosts
```

Comment:
Do you plan to add docker and open stack deployment back? Or was it not working originally?